### PR TITLE
[Do not merge] Federator logs

### PR DESCRIPTION
**Pull request by RafilxTenfen**

### CODE REVIEW

File: federator/integrationTest/integrationTest.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ function getFederators(configFile, keys, storagePathPrefix = "fed") {
   const federators = [];
   if (keys && keys.length) {
     keys.forEach((key, i) => {
-      const federator = new Federator(
+      const federator = new Federator.Federator(
```
Comment (pmprete) [inline federator/integrationTest/integrationTest.js:64]: this looks a bit odd

File: federator/config/types.ts [hunk 2]
```diff
@@ -1,24 +0,0 @@
-export class ConfigChain {
```
Comment (pmprete) [inline federator/config/types.ts:1]: Why did we remove the types?

File: federator/src/lib/Federator.ts [hunk 3]
```diff
@@ -1,52 +1,55 @@
...
+import fs from 'fs';
+import TransactionSender from './TransactionSender';
+import CustomError from './CustomError';
+import { BridgeFactory } from '../contracts/BridgeFactory';
+import { FederationFactory } from '../contracts/FederationFactory';
+import { AllowTokensFactory } from '../contracts/AllowTokensFactory';
+import utils from './utils';
+import * as typescriptUtils from './typescriptUtils';
+
+export class Federator {
```
Comment (pmprete) [inline federator/src/lib/Federator.ts:15]: export as default to avoid the Federator.Federator thingy

**Review by pmprete — COMMENTED** [review]
Except my comments looks good, please review the tests that are failing in CI

File: federator/src/lib/Federator.ts (hunk 3 above)
Comment (RafilxTenfen) [reply federator/src/lib/Federator.ts:15]: oww, nice!

File: federator/config/types.ts (hunk 2 above)
Comment (RafilxTenfen) [reply federator/config/types.ts:1]: I pute it in another file `lib/config.ts`

**Review by pmprete — APPROVED** [review]
LGTM

**Pull request merged**